### PR TITLE
sshd: add page

### PR DESCRIPTION
**Pull request by nobiit**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/sshd.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# sshd
+
+> Secure Shell Daemon - allows remote machines to securely log in to the current machine.
+> Remote machines can execute commands as it is executed at this machine.
+> More information: <https://linux.die.net/man/8/sshd>.
```
Comment (navarroaxel) [inline pages/common/sshd.md:5]: I think we should use the official manual:
```suggestion
> More information: <https://man.openbsd.org/sshd>.
```

Comment (nobiit) [reply pages/common/sshd.md:5]: Thanks

**Review by owenvoke — APPROVED** [review]

File: pages/common/sshd.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/sshd.md:5]: @nobiit but you didn't change the link.

Comment (nobiit) [reply pages/common/sshd.md]: Oh. I forgot, thank you again

File: pages/common/sshd.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# sshd
+
+> Secure Shell Daemon - allows remote machines to securely log in to the current machine.
+> Remote machines can execute commands as it is executed at this machine.
+> More information: <https://man.openbsd.org/sshd>.
+
+- Start Daemon and run in the background:
```
Comment (navarroaxel) [inline pages/common/sshd.md:7]:
```suggestion
- Start daemon in the background:
```

File: pages/common/sshd.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+`sshd`
+
+- Start Daemon and keep foreground:
```
Comment (navarroaxel) [inline pages/common/sshd.md:11]: It's not a daemon if it's in foreground. Right?
```suggestion
- Run sshd in the foreground:
```

File: pages/common/sshd.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+`sshd -D`
+
+- Start Daemon and show logging details:
```
Comment (navarroaxel) [inline pages/common/sshd.md:15]: Perhaps this better
```suggestion
- Run with verbose output (for debugging):
```

File: pages/common/sshd.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+`sshd -D -d`
+
+- Run on another port:
```
Comment (navarroaxel) [inline pages/common/sshd.md:19]:
```suggestion
- Run on a specific port:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/sshd.md (hunk 5 above)
Comment (nobiit) [reply pages/common/sshd.md]: Thanks @navarroaxel

**Review by navarroaxel — APPROVED** [review]

**Comment by nobiit** [comment]
Thanks, @navarroaxel and @owenvoke

**Pull request merged**